### PR TITLE
[WIP]: GNOME 43

### PR DESCRIPTION
**Pull request by paper42**
~~very WIP, many shlibs are not resolved, many packages need to be revbumped, many packages updated, many commits have to be improved, etc. Feel free to test this PR, but expect to have to fix a few things at this point.~~

depends on:
- [x] https://github.com/void-linux/void-packages/pull/39488
- [x] https://github.com/void-linux/void-packages/pull/39498
- [x] https://github.com/void-linux/void-packages/pull/39515
- [x] https://github.com/void-linux/void-packages/pull/39514
- [x] https://github.com/void-linux/void-packages/pull/40701
- [x] https://github.com/void-linux/void-packages/pull/40702
- [x] https://github.com/void-linux/void-packages/pull/41019
- [x] https://github.com/void-linux/void-packages/pull/41020
- [ ] https://github.com/void-linux/void-packages/pull/41021
- [x] https://github.com/void-linux/void-packages/pull/41022
- [x] https://github.com/void-linux/void-packages/pull/41023
- [x] https://github.com/void-linux/void-packages/pull/41024
- [x] https://github.com/void-linux/void-packages/pull/41026

### CODE REVIEW

File: srcpkgs/gnome-desktop/template [hunk 1]
```diff
@@ -15,9 +15,10 @@ short_desc="GNOME desktop management utilities"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://gitlab.gnome.org/GNOME/gnome-desktop"
-changelog="https://gitlab.gnome.org/GNOME/gnome-desktop/-/raw/gnome-42/NEWS"
+#changelog="https://gitlab.gnome.org/GNOME/gnome-desktop/-/raw/gnome-43/NEWS"
+chabgelog="https://gitlab.gnome.org/GNOME/gnome-desktop/-/raw/master/NEWS"
```
Comment (oreo639) [inline srcpkgs/gnome-desktop/template:19]: Typo

File: srcpkgs/xdg-desktop-portal-gnome/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'xdg-desktop-portal-gnome'
 pkgname=xdg-desktop-portal-gnome
 version=42.3
```
Comment (oreo639) [inline srcpkgs/xdg-desktop-portal-gnome/template:3]: xdg-desktop-portal-gnome 43.0 is avaliable. (will require xdg-desktop-portal 1.15)

Comment (paper42) [reply srcpkgs/xdg-desktop-portal-gnome/template:3]: Can we update it separately or does it need gnome 43?

Comment (oreo639) [reply srcpkgs/xdg-desktop-portal-gnome/template:3]: Ah, you're right, my bad.

File: srcpkgs/gnome-desktop/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/gnome-desktop/template]: fixed

File: srcpkgs/gdm/template [hunk 3]
```diff
@@ -23,7 +23,7 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/GDM"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=a328ac976937bec179971ff07829a1246b98ce724181c128ae32a0ba47ad3855
+checksum=94d70d6ed7df59fa7fde4fd02fed1168593a8adce10fcee113c148d52b392294
 
 conf_files="
```
[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/gdm/template:29]: applied, thanks

File: srcpkgs/gnome-console/template [hunk 4]
```diff
@@ -1,26 +1,23 @@
...
-configure_args="-Dnautilus=enabled"
-hostmakedepends="pkg-config desktop-file-utils gettext sassc
+hostmakedepends="pkg-config desktop-file-utils gettext
  gobject-introspection glib-devel gtk-update-icon-cache"
-makedepends="libhandy1-devel vte3-devel libgtop-devel nautilus-devel
+makedepends="libadwaita-devel vte3-gtk4-devel libgtop-devel
  gsettings-desktop-schemas-devel pcre2-devel"
 depends="gsettings-desktop-schemas"
 short_desc="Simple user-friendly terminal emulator for the GNOME desktop"
 maintainer="oreo639 <oreo6391@gmail.org>"
```
Comment (oreo639) [inline srcpkgs/gnome-console/template:12]: Can you change this to oreo639@gmail.com.

File: srcpkgs/gcr4/template [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Template file for 'gcr4'
+pkgname=gcr4
+version=3.92.0
```
Comment (oreo639) [inline srcpkgs/gcr4/template:3]: gcr 4.0.0 is out now https://gitlab.gnome.org/GNOME/gcr/-/tree/4.0.0

File: srcpkgs/gnome-initial-setup/template [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
 pkgname=gnome-initial-setup
-version=42.2
-revision=2
+version=43.0
+revision=1
 build_style=meson
 configure_args="-Dparental_controls=disabled -Dsystemd=false"
 hostmakedepends="pkg-config glib-devel gettext"
 makedepends="NetworkManager-devel accountsservice-devel libglib-devel
  gnome-desktop-devel cheese-devel libgweather-devel webkit2gtk-devel
```
Comment (oreo639) [inline srcpkgs/gnome-initial-setup/template:9]: I think this is supposed to be libwebkit2gtk50-devel

**Comment by oreo639** [comment]
gnome-terminal needs to be updated for nautilus:
https://gist.github.com/oreo639/eebf05ac651c77670bd0270e8f7acf16
(I haven't tested it yet though)
Also, xdg-desktop-portal-gnome needs to be revbumped.

File: srcpkgs/GPaste/template [hunk 7]
```diff
@@ -1,23 +1,25 @@
 # Template file for 'GPaste'
 pkgname=GPaste
-version=42.1
+version=43.0
 revision=1
 build_helper="gir"
 build_style=meson
 configure_args="$(vopt_bool gnome_shell gnome-shell)
  -Dcontrol-center-keybindings-dir=/usr/share/gnome-control-center/keybindings
  -Dsystemd-user-unit-dir=/usr/lib/systemd"
```
Comment (oreo639) [inline srcpkgs/GPaste/template:9]: This should probably be `/usr/lib/systemd/user`

Comment (paper42) [reply srcpkgs/GPaste/template:9]: thanks, done in 14a20ebadfe5e4db3c7f913c07ce20f44f3b1b37

**Comment by paper42** [comment]
merged

**Comment by motorto** [comment]
Thanks for the hard work @paper42 and everyone else involved (@oreo639 , and all the others) !!!

**Comment by thegarlynch** [comment]
Do we disabled device security settings like ubuntu too ?

**Comment by paper42** [comment]
> Do we disabled device security settings like ubuntu too ?

It was not disabled.

**Comment by thegarlynch** [comment]
seems like do to chassis system-d hostname system. it become invisible for void ? 
https://www.reddit.com/r/gnome/comments/xc2ryq/gnome_43_device_security_page_missing/
https://gitlab.gnome.org/GNOME/gnome-control-center/-/commit/0245329949957e7f7ab56f240faaf336408ced86

**Comment by lane-brain** [comment]
I think this update has plunged my system into some shlib hell, just wanted to note it for anyone else having the same issue
```
[*] Updating repository `https://repo-default.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
gnome-calendar-42.2_2: broken, unresolvable shlib `libecal-2.0.so.1'
gnome-shell-42.4_1: broken, unresolvable shlib `libecal-2.0.so.1'
gnome-calendar-42.2_2: broken, unresolvable shlib `libedataserver-1.2.so.26'
gnome-shell-42.4_1: broken, unresolvable shlib `libedataserver-1.2.so.26'
gnome-control-center-42.3_1: broken, unresolvable shlib `libgnome-bg-4.so.1'
xdg-desktop-portal-gnome-43.1_1: broken, unresolvable shlib `libgnome-bg-4.so.1'
gnome-session-42.0_2: broken, unresolvable shlib `libgnome-desktop-3.so.19'
gnome-settings-daemon-42.2_2: broken, unresolvable shlib `libgnome-desktop-3.so.19'
gnome-shell-42.4_1: broken, unresolvable shlib `libgnome-desktop-3.so.19'
mutter-42.4_1: broken, unresolvable shlib `libgnome-desktop-3.so.19'
gnome-clocks-42.0_2: broken, unresolvable shlib `libgnome-desktop-4.so.1'
gnome-control-center-42.3_1: broken, unresolvable shlib `libgnome-desktop-4.so.1'
gnome-initial-setup-42.2_2: broken, unresolvable shlib `libgnome-desktop-4.so.1'
xdg-desktop-portal-gnome-43.1_1: broken, unresolvable shlib `libgnome-desktop-4.so.1'
gnome-control-center-42.3_1: broken, unresolvable shlib `libgnome-rr-4.so.1'
```

**Comment by JamiKettunen** [comment]
@lane-brain what command exactly did you try to run, `xbps-install -Su` or?

**Comment by lane-brain** [comment]
> @lane-brain what command exactly did you try to run, `xbps-install -Su` or?

I had always assumed that `xi -u` was equivalent to `xbps-install -Su`, but after running the normal command I was able to update my packages. Thanks.

**Pull request closed without merging**